src/games/strategy/performance/Perf.java

```diff
@@ -1,56 +0,0 @@
-package games.strategy.performance;
-
-import java.util.prefs.Preferences;
-
-
-/**
- * Provides a high level API to the game engine for performance measurements.
- * This class handles the library details and sends output to 'PerformanceConsole.java'
- */
-// TODO Move the methods in this class to PerfTimer
-public class Perf {
```

Comment:
sure enough.  It looks to still be auto-closeable now. Will review tomorrow, after a quick look things looked okay.